### PR TITLE
LPD-39475 Remove com.liferay.headless.batch.engine.resource.v1_0.test.ExportTaskResourceTest from acceptance

### PR DESCRIPTION
**Pull request by GaborKomaromi**

### CODE REVIEW

**Review by magjed4289 — APPROVED** [review]
Let's try! :D The path seems to be correct, let's see if CI can handle it ;)

**Comment by magjed4289** [comment]
ci:forward

File: test.properties [hunk 1]
```diff
@@ -3592,6 +3592,7 @@
     test.batch.class.names.excludes[acceptance]=\
         **/batch-planner/**/*Test.java,\
         **/headless-builder/**/*Test.java,\
+        **/object-rest-test/**/ExportTaskResourceTest.java,\
```
Comment (4lejandrito) [inline test.properties:3595]: @GaborKomaromi wrong test. This is the right one:

```
com.liferay.headless.batch.engine.resource.v1_0.test.ExportTaskResourceTest
```

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

**Comment by 4lejandrito** [comment]
Guys see my comments. I just closed Brian's pr.

File: test.properties (hunk 1 above)
Comment (magjed4289) [reply test.properties:3595]: oh shhh. didn't realise that, thanks for noticing @4lejandrito!

Comment (GaborKomaromi) [reply test.properties]: Changed to the right one.

**Comment by GaborKomaromi** [comment]
Hey sorry for the mistake. Changed to the correct test.

**Comment by GaborKomaromi** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**